src/www/system_advanced_firewall.php

```diff
@@ -43,7 +43,6 @@
     $pconfig['maximumfrags'] = isset($config['system']['maximumfrags']) ? $config['system']['maximumfrags'] : null;
     $pconfig['adaptivestart'] = isset($config['system']['adaptivestart']) ? $config['system']['adaptivestart'] : null;
     $pconfig['adaptiveend'] = isset($config['system']['adaptiveend']) ? $config['system']['adaptiveend'] : null;
-    $pconfig['noantilockout'] = isset($config['system']['webgui']['noantilockout']);
```

Comment:
Not sure if I said this earlier, but it's not a good idea to aim for a replacement at this point without a migration strategy.

Reply:
Ah, now I got what you meant. You still want to keep a global on/off switch for the whole feature. Yes, that makes sense.

So I have updated the PR once again and now the changes are as minimal as possible:
Interfaces can have an attribute "antilockout". Interfaces with this attribute take precedence when trying to find interfaces to activate the anti lockout rule for. If no such interfaces are found, the old (buggy) logic applies.

If the "Disable anti-lockout" setting in Firewall/Settings/Advanced is checked, no anti lockout rules will be generated, regardless of the new attribute.

Resetting to defaults (or generating the default configuration) will set the new antilockout attribute on the LAN interface.

This way there is also no need to migrate anything for now.

This doesn't fully solve the issue, but it makes it even less likely and now there is actually an option to define the interfaces for which the anti lockout rule shall apply for.